src/app/shell/personal-cabinet/shared-cabinet/applications/application-card/info-status/info-status.component.ts

```diff
@@ -32,14 +33,23 @@ export class InfoStatusComponent implements OnInit, OnDestroy {
 
   public status: ApplicationStatuses | UserStatuses.Blocked;
   public reason: string;
+  public competitiveSelectionDescription: string;
 
   private destroy$: Subject<boolean> = new Subject<boolean>();
 
-  constructor(private store: Store) {}
+  constructor(
+    private store: Store,
+    private workshopService: UserWorkshopService
+  ) {}
 
   public ngOnInit(): void {
     this.status = this.application.isBlockedByProvider ? UserStatuses.Blocked : ApplicationStatuses[this.application.status];
     this.reason = !this.application.isBlockedByProvider && this.application.rejectionMessage;
+    if (this.status && this.status.toString() === 'AcceptedForSelection') {
```

Comment:
I suggest to use [enum](https://github.com/ita-social-projects/OoS-Frontend/blob/72fe3d8f03b6b1c427812979b63eab9c720850a4/src/app/shared/enum/statuses.ts#L27) here
```suggestion
    if (this.status === ApplicationStatuses.AcceptedForSelection) {
```